apps/dashboard/app/(app)/mobile-sidebar.tsx

```
@@ -30,36 +45,32 @@ export const MobileSideBar = ({ className }: Props) => {
-                </Button>
-              </Link>
-              <SignOutButton signOutCallback={() => router.push("/auth/sign-in")}>
-                <SheetClose asChild>
-                  <Button variant="ghost" className="justify-start w-full py-2 border-t">
-                    <LogOut className="w-4 h-4 mr-2" />
-                    Sign Out
-                  </Button>
-                </SheetClose>
-              </SignOutButton>
+              {workspaceNavigation.map((item) => (
+                <Link href={`${item.href}`}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)